chia/cmds/configure.py

```diff
@@ -260,6 +270,12 @@ def configure(
     help="configures the seeder nameserver setting. Ex: `example.com.`",
     type=str,
 )
+@click.option(
+    "--enable-data-server",
+    "--data-server",
```

Comment:
I see that `click.Choice` accepts any argument in the array.
`--enable-data-server false` is more clear to me than `--data-server false`
I would only keep one of them.

If you wanted to keep both, it might be good to have `--enable-data-server` and `--disable-data-server`

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/13222